### PR TITLE
feat: add event poster card datalization render

### PR DESCRIPTION
**Pull request by bzp2010**
Changes:

Remove 10.23 meetup ad on index page.

### CODE REVIEW

**Review by iamayushdas — APPROVED** [review]
Just check the failing build, otherwise its fine

File: website/static/data/event-poster-card.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "show": false,
+  "expire": "2021-10-25",
+  "image": "/blog/2021/10/18/meetup",
+  "link": "https://static.apiseven.com/202108/meetup-card.png"
```
Comment (juzhiyuan) [inline website/static/data/event-poster-card.json]: 🤔

File: website/static/data/event-poster-card.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "show": false,
+  "expire": "2021-10-25",
```
Comment (juzhiyuan) [inline website/static/data/event-poster-card.json:3]: Why do we need both show & expire?

Comment (1502shivam-singh) [reply website/static/data/event-poster-card.json:3]: Same, `expire` field is understandable, but what's `show` field for ? Some sort of safeguard is it ?

File: website/static/data/event-poster-card.json (hunk 1 above)
Comment (1502shivam-singh) [reply website/static/data/event-poster-card.json]: I like this style, all changes needed are self explanatory and at one place and don't need codebase changes, so someone not involved much with the codebase can also do this. Not sure about the `show` field though.

File: website/static/data/event-poster-card.json (hunk 2 above)
Comment (bzp2010) [reply website/static/data/event-poster-card.json:3]: @juzhiyuan, @1502shivam-singh The `show` overwrites anything that does not exceed the expiration time but does not want to be displayed.

**Review by juzhiyuan — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

File: website/static/data/event-poster-card.json (hunk 1 above)
Comment (bzp2010) [reply website/static/data/event-poster-card.json]: fixed

**Review by LiteSun — APPROVED** [review]

**Pull request merged**